streams-contrib/streams-persist-cassandra/src/main/jsonschema/org/apache/streams/cassandra/CassandraConfiguration.json

```diff
@@ -0,0 +1,44 @@
+{
+  "$schema": "http://json-schema.org/draft-03/schema",
+  "$license": [
+    "http://www.apache.org/licenses/LICENSE-2.0"
+  ],
+  "id": "#",
+  "type": "object",
+  "javaType": "org.apache.streams.cassandra.CassandraConfiguration",
+  "javaInterfaces": ["java.io.Serializable"],
+  "properties": {
+    "host": {
+      "type": "string",
```

Comment:
perhaps this should be an array of strings, so module can start-up with more than one seed host.